src/videojs-contrib-hls.js

```diff
@@ -58,6 +58,98 @@ Object.defineProperty(Hls, 'GOAL_BUFFER_LENGTH', {
   }
 });
 
+Object.defineProperty(Hls, 'MAX_GOAL_BUFFER_LENGTH', {
```

Comment:
I would also add that you could shove the property string 'MAX_GOAL_BUFFER_LENGTH' into a variable somewhere.

If these functions are all similar enough you could probably dump them all in an array and run through it to define these objects.